sdk/documentintelligence/azure-ai-documentintelligence/samples/async_samples/sample_manage_models_async.py

```diff
@@ -62,12 +62,13 @@ async def sample_manage_models():
             print("Doc types the model can recognize:")
             for name, doc_type in model.doc_types.items():
                 print(f"Doc Type: '{name}' built with '{doc_type.build_mode}' mode which has the following fields:")
-                for field_name, field in doc_type.field_schema.items():
-                    if doc_type.field_confidence:
-                        print(
-                            f"Field: '{field_name}' has type '{field['type']}' and confidence score "
-                            f"{doc_type.field_confidence[field_name]}"
-                        )
+                if doc_type.field_schema:
```

Comment:
Did this become optional in this version?

Reply:
Yes. https://github.com/Azure/azure-sdk-for-python/blob/3f96dcf6270ffac8492b38b8d3bb338a0e39be54/sdk/documentintelligence/azure-ai-documentintelligence/CHANGELOG.md?plain=1#L36

Reply:
Ah I missed that, thanks for point it out!